lib/galaxy/workflow/resources/example.py

```diff
@@ -0,0 +1,22 @@
+import logging
```

Comment:
Can you ``git rm`` this file from the git index - I guess only the sample should be in here.